control_panel_api/views.py

```diff
@@ -74,32 +110,65 @@ class AppS3BucketViewSet(viewsets.ModelViewSet):
 
     def perform_create(self, serializer):
         apps3bucket = serializer.save()
-        apps3bucket.aws_create()
+
+        try:
+            apps3bucket.aws_create()
+        except ClientError as e:
+            logger.error(e)
+            raise AWSException
 
     def perform_update(self, serializer):
         apps3bucket = serializer.save()
-        apps3bucket.aws_update()
+
+        try:
+            apps3bucket.aws_update()
+        except ClientError as e:
+            logger.error(e)
+            raise AWSException
 
     def perform_destroy(self, instance):
```

Comment:
Does this method needs to be wrapped in a transaction?

Reply:
👍